platforms/hyperledger-besu/configuration/samples/network-besu.yaml

```diff
@@ -14,6 +14,10 @@ network:
   type: besu
   version: 21.10.6  #this is the version of Besu docker image that will be deployed.
 
+  # providing letsencrypt true would create tls certificate using letsencrypt, 
+  #This is Optional, if no value or false provided, self signed certificates will be created
+  letsencrypt: false
```

Comment:
please don't be specific to `letsencrypt`, you can do this based on the issuer, where we can define any issuer and certificates can be provisioned by that.

Reply:
as of now cert manager supports either letsencrypt or selfsigned certificates

Reply:
cert manager supports other issuer as well like CA, vault, Venafi etc. Ref: https://cert-manager.io/docs/configuration/